FusionIIIT/templates/department/sm_dep.html

```diff
@@ -108,15 +129,38 @@ <h3>
                                 </p>
                             </li>
                         </ol>
-                    </span>
+                    </span> {% endcomment %}
```

Comment:
If you are making this part of code as comment, kindly remove the code and don't make it as comment.